app/ports/models/port.py

```diff
@@ -313,3 +313,8 @@ class Meta:
             models.Index(fields=['github']),
             models.Index(fields=['name', 'domain'])
         ]
+
+
+class Commit(models.Model):
+    hash = models.CharField(max_length=50)
+    updated_at = models.DateTimeField(auto_now=True)
```

Comment:
Moving this part is ok. What's confusing though is that you'll probably store just the latest commit, right? Or will you store more? If you only store the latest commit, the name should be slightly more descriptive. No idea about the best one, but maybe something in the direction of "LastPortInfoUpdate" class with `git_commit_hash` as the first field.

Reply:
I was earlier thinking of storing all (when running updates every 12 hours), but if we run them more frequently then they will just eat space in the database. I will change the functionality and names. Thanks

Reply:
This has been fixed: ea2eccbaf84e98737dd696e5fc9e0c8c9eb7f7c8